x-pack/plugins/search_playground/public/components/message_list/assistant_message.tsx

```diff
@@ -53,53 +53,78 @@ export const AssistantMessage: React.FC<AssistantMessageProps> = ({ message }) =
   return (
     <>
       {!!retrievalDocs?.length && (
-        <EuiComment
-          username={username}
-          timelineAvatar="dot"
-          data-test-subj="retrieval-docs-comment"
-          eventColor="subdued"
-          css={{
-            '.euiAvatar': { backgroundColor: euiTheme.colors.ghost },
-            '.euiCommentEvent': {
-              border: euiTheme.border.thin,
-              borderRadius: euiTheme.border.radius.medium,
-            },
-          }}
-          event={
-            <>
+        <>
+          <EuiComment
+            username={username}
+            timelineAvatar="dot"
+            data-test-subj="assistant-message-searching"
+            eventColor="subdued"
+            css={{
+              '.euiAvatar': { backgroundColor: euiTheme.colors.ghost },
+              '.euiCommentEvent': {
+                border: euiTheme.border.thin,
+                borderRadius: euiTheme.border.radius.medium,
+              },
+            }}
+            event={
               <EuiText size="s">
                 <p>
                   <FormattedMessage
-                    id="xpack.searchPlayground.chat.message.assistant.retrievalDocs"
-                    defaultMessage="Grounding answer based on"
+                    id="xpack.searchPlayground.chat.message.assistant.searchingQuestion"
+                    defaultMessage='Searching "{question}"'
+                    values={{ question: inputTokens.searchQuery }}
                   />
-                  {` `}
```

Comment:
can you keep this space for this 

![Screenshot 2024-08-05 at 14 34 33](https://github.com/user-attachments/assets/5a61706b-9a89-426c-9097-19a5fc80c1a2)

Reply:
Replaced with adding a space at the end of sentence in 4ab7ae9ee4337a6ac81ca03772b99cd7352ecbb4. I think this would add the space . With this I think we can avoid adding extra space in next line but rather included as part of sentence?

Reply:
The challenge here is if the translation strings starts to remove the space

Reply:
Reverted to original in https://github.com/elastic/kibana/commit/6a35192889611718e9edceffe4dde1f47359bc57